__tests__/index.spec.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import { format, render, cancel, register } from '../src/';
+import { getLocale } from '../src/register';
```

Comment:
I have renamed locales.ts because register better reflects the content and follows similar naming to format.ts